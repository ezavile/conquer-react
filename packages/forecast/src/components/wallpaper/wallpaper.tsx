/** @jsx jsx */
import { css, jsx } from '@emotion/core';
import React from 'react';

interface Wallpaper {
  url: string;
}

export const Wallpaper: React.FC<Wallpaper> = ({ url }) => (
  <div
    className="h-screen bg-cover bg-center"
    css={css`
      background-color: #000;
      box-shadow: inset 0 0 0 100vw rgba(0, 0, 0, 0.5);
      background-image: url(${url});
    `}
  />
);
