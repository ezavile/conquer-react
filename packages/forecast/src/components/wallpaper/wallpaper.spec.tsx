import React from 'react';

import { render } from '@testing-library/react';
import { Wallpaper } from './wallpaper';

describe('wallpaper', () => {
  it('renders', () => {
    const { asFragment } = render(<Wallpaper url="algo" />);

    expect(asFragment()).toMatchSnapshot();
  });
});
