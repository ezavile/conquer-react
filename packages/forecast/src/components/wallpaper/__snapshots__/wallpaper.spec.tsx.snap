// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`wallpaper renders 1`] = `
<DocumentFragment>
  <div
    class="h-screen bg-cover bg-center css-1l26nll-Wallpaper"
  />
</DocumentFragment>
`;
